src/main/resources/mixins.iris.json

```diff
@@ -4,16 +4,17 @@
   "package": "net.coderbot.iris.mixin",
   "compatibilityLevel": "JAVA_8",
   "client": [
-	"MixinGameRenderer",
-	"MixinImmediateVertexConsumerProvider",
-	"MixinWorldRenderer",
-	"texunits.MixinBufferVertexConsumer",
-	"texunits.MixinGlStateManager",
-	"texunits.MixinLightmapTextureManager",
-	"texunits.MixinOverlayTexture",
-	"texunits.MixinVertexFormats"
+    "MixinGameRenderer",
+    "MixinImmediateVertexConsumerProvider",
+    "MixinWorldRenderer",
+    "texunits.MixinBufferVertexConsumer",
+    "texunits.MixinGlStateManager",
+    "texunits.MixinLightmapTextureManager",
+    "texunits.MixinOverlayTexture",
+    "texunits.MixinVertexFormats",
+    "MixinTranslationStorage"
```

Comment:
Everytime I go into this file, intellij just reformats the whole file, I'm not sure why

Reply:
It looks like the indentation on the file was messed before up so this is probably okay